sceptre/resolvers/project_variables.py

```diff
@@ -24,6 +26,12 @@ def resolve(self):
         :returns: Value at argument location.
         :rtype: str
         """
+        init(autoreset=True)
+        simplefilter('always', DeprecationWarning)
+        warn(Fore.YELLOW + (
+                "The project_variables resolver is being deprecated, and will "
```

Comment:
The project_variables is ~~being~~ deprecated, and will be remove in later versions. (I don't want to make any guarantees of which version it will go.) Maybe also nice to offer an alternative solution to using project_variables. - but know its tricky as its own purpose is unclear. Possibly use --vars instead?